### ISSUE TITLE
[Suggestion]: однакові назви стандартних пунктів для reference-сторінок

### ISSUE THREAD

**Issue by alinkedd** [reactjs/uk.react.dev#567]
### Summary

Необхідно перевірити після завершення основних перекладів, щоб усі reference-сторінки мали однакові назви стандартних пунктів.

### Details

Поточні назви:
Reference - Опис
Parameters - Параметри
Returns - Результат
Caveats - Застереження
Usage - Використання
Troubleshooting - Усунення несправностей

**Comment by pavlosambur** [comment]
**Returns - Результат**... "Повернення" не краще? Ось, наприклад, у Вікіпедії [Тип повернення значення функції](https://uk.wikipedia.org/wiki/%D0%A2%D0%B8%D0%BF_%D0%BF%D0%BE%D0%B2%D0%B5%D1%80%D0%BD%D0%B5%D0%BD%D0%BD%D1%8F_%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%8F_%D1%84%D1%83%D0%BD%D0%BA%D1%86%D1%96%D1%97). Перекладати "return" як "результат" зовсім неправильно. 

І варіант **Troubleshooting - Усунення несправностей** обговорюється? Калька з англійської. 
Як на мене в контексті програмування Troubleshooting як назву розділа краще перекладати як "Поширені проблеми".
У більшості пунктів цього розділу описано ситуації типу "я наче все робив правильно, але щось не працює..." Тобто не йдеться про несправності в технічному сенсі, а саме про типові баги, помилки або складні для розуміння ситуації. "Несправності" - це якщо щось не заводиться і не їде. Якщо не працює код ніхто не каже "у мене несправність у функції". Натомість "виникла проблема / десь бага / щось пішло не так / не працює як слід". Як альтернатива, можна цей розділ назвати "Якщо щось пішло не так".

**Comment by alinkedd** [comment]
@pavlosambur щодо returns, то тут або `повертає`, або те, що повертається, а не процес повернення. тому вибрали результат, як те, що повертається, щоб не використовувати дієслово. можу погодитися з "Повертає", але не з "Повернення".

щодо troubleshooting, то просто прямий переклад. ну і ми намагаємося прямо перекладати або близько до значення. як тобі тоді "Усунення проблем"?

і я б, мабуть, ще Reference як Довідка переклала б

**Comment by pavlosambur** [comment]
"Returns" я пропоную "Повернення" не як дієщось, а саме як іменник, що позначає те, що функція повертає як результат своєї роботи. Це не процес, не дія - це сутність.

Ось наприклад.
Ця функція виконує обчислення, але не має значення повернення.
Тип повернення цієї функції — number.

Відчуваєш? ;)

Або в побуті ми можемо сказати щось ніби.
"Я оформив повернення," чи "повернення здійснюється в термін до трьох робочих днів", тощо.

Ти пишеш, що вибрали таке, щоб не було дієсловом і зразу пропонуєш "повертає".  Returns в нас тут не дієслово, а ТЕ, що може бути тим, що функція повертає чи не повертає. І ось це ТЕ - є її поверненням.

Гуглив, вивчав, спілкувався з GPT. Ось зараз ще з тобою спілкуюсь - ще вивчаю. Шукаю аргументи за "повертає". Вибач, не находжу. Термін "повернення" - вже вкорінений у технічній українській мові.

**Comment by pavlosambur** [comment]
Усунення "проблем" значно краще ніж "несправностей".
Прекрасний варіант!

**Comment by pavlosambur** [comment]
Reference як Довідка переклала б
----
Доречі, "опис" тут не ідеально... Згоден. 
В цих розділах наводиться повний перелік параметрів сутностей, їх опис, можливі значення і так далі. Це не опис.
Пропоную "Довідник". Бо "довідка" трошки побутове наче, наче те, що на вокзалі.

**Comment by alinkedd** [comment]
@pavlosambur 

> Ти пишеш, що вибрали таке, щоб не було дієсловом і зразу пропонуєш "повертає". Returns в нас тут не дієслово, а ТЕ, що може бути тим, що функція повертає чи не повертає. І ось це ТЕ - є її поверненням.

> Гуглив, вивчав, спілкувався з GPT. Ось зараз ще з тобою спілкуюсь - ще вивчаю. Шукаю аргументи за "повертає". Вибач, не находжу. Термін "повернення" - вже вкорінений у технічній українській мові.

Так, мені ближче "Повертає", але шоб не було дієслова, то вибрали "Результат". Вживається і "результат функції", "тип результату функції".

"Повернення" - це не те, що вертається, це процес вертання, це схоже з англійським герундієм на -ing, тобто returning. І на тій же [вікіпедії](https://uk.wikipedia.org/wiki/%D0%A2%D0%B8%D0%BF_%D0%BF%D0%BE%D0%B2%D0%B5%D1%80%D0%BD%D0%B5%D0%BD%D0%BD%D1%8F_%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%8F_%D1%84%D1%83%D0%BD%D0%BA%D1%86%D1%96%D1%97) є "значення повернення" і "результат" як синоніми, тобто треба тоді писати або "значення повернення" (два слова, не лише "повернення"), або "результат", і "результат" є коротшим і компактнішим, тому я за результат.

> Усунення "проблем" значно краще ніж "несправностей".

Тоді зміню на усунення проблем

> Доречі, "опис" тут не ідеально... Згоден.
В цих розділах наводиться повний перелік параметрів сутностей, їх опис, можливі значення і так далі. Це не опис.
Пропоную "Довідник". Бо "довідка" трошки побутове наче, наче те, що на вокзалі

Можна ще погратися зі словом "сигнатура", але якраз тут "довідка" - це опис кожної сутності, щоб дізнатися базову функціональність і сигнатуру. "Довідником" тут є уся секція Reference, бо це якраз як книга (reference book). А тут саме опис конкретної функції, треба ще подумати
